compiler-rt/test/asan/TestCases/interception_failure_test.cpp

```diff
@@ -18,6 +18,10 @@
 // it works with the dynamic runtime.
 // XFAIL: target={{.*netbsd.*}} && !asan-dynamic-runtime
 
+#if _MSC_VER && !defined(__clang__)
```

Comment:
`defined(_MSC_VER)` (Also please grep for more cases of this in the patch, I probably won't comment on all of them.)

Reply:
done and I've also grepped for relevant cases, I avoided changing anything outside of compiler-rt